### ISSUE TITLE
argument customer_id is missing @ list_customer_credit_balances()

### ISSUE THREAD

**Issue by Vincent17201** [websideproject/paddle-billing-client#278]
## 🐛 Bug Report

argument customer_id is missing @ list_customer_credit_balances()

## 🔬 How To Reproduce

call list_customer_credit_balances()

### Code sample

quick fix:

```
def list_customer_credit_balances(
        self, customer_id: str, query_params: CustomerBalancesQueryParams = CustomerBalancesQueryParams()
    ) -> CustomerBalancesResponse:
        """List credit balances for a customer"""
        return self.get(
            self.endpoints.list_customer_credit_balances.format(customer_id=customer_id),
            params=query_params.model_dump(exclude_none=True),
        )
```
### Screenshots

<img width="921" alt="image" src="https://github.com/websideproject/paddle-billing-client/assets/90750980/00674ad0-3c48-47dc-8f33-69f7819ec4f7">

## 📈 Expected behavior
get balances for a given customer id

**Comment by bgervan** [comment]
Thanks for the report. I have added the fix and created a version with it:
https://github.com/websideproject/paddle-billing-client/releases/tag/v0.2.7

**Comment by bgervan** [comment]
I had issues with poetry-publish, so it is fixed and published with v0.2.9 only

**Comment by Vincent17201** [comment]
@bgervan: Great, many thanks for the quick merge 👍
